lib/ansible/modules/cloud/cloudstack/cs_instance_info.py

```diff
@@ -43,17 +46,28 @@
 '''
 
 EXAMPLES = '''
-- name: gather instance facts
-  cs_instance_facts:
+- name: gather instance information
+  cs_instance_info:
     name: web-vm-1
   delegate_to: localhost
   register: vm
 
-- debug:
-    var: cloudstack_instance
-
-- debug:
+- name: Show the returned results of the registered variable
+  debug:
     var: vm
```

Comment:
I'm a broken record, sorry
```suggestion
     msg: "{{ vm }}"
```
Could be done below and for cs_zone_info too.

Reply:
Thanks, changed it.